docs/lib/auth/fragments/js/advanced.md

```diff
@@ -1,3 +1,15 @@
+## Back Doors
+
+Depending on your use case, you may want to utilize either [`amazon-cognito-identity-js`](https://github.com/aws-amplify/amplify-js/tree/main/packages/amazon-cognito-identity-js) (the User Pool client, which underlies Amplify Auth), [`aws-sdk-js`](https://github.com/aws/aws-sdk-js), or [`aws-sdk-v3-js`](https://github.com/aws/aws-sdk-js-v3) directly.
+
+### `amazon-cognito-identity-js`
+
+The [`amazon-cognito-identity-js`](https://github.com/aws-amplify/amplify-js/tree/main/packages/amazon-cognito-identity-js) package is responsible for forming AWS requests for common User Pool actions, while abstracting away the need to manually persist sessions and tokens between those requests. It also computes [SRP](https://en.wikipedia.org/wiki/Secure_Remote_Password_protocol) values to secure your authentication and challenge requests. This package is maintained with [the `amplify-js` repository](https://github.com/aws-amplify/amplify-js) (copied from [its original repository](https://github.com/amazon-archives/amazon-cognito-identity-js), which has since been deprecated and moved into [`amazon-archives`](https://github.com/amazon-archives)).
```

Comment:
The provenance of the code in terms of repo doesn't seem relevant in a documentation context. Is there documentation about this underlying code you can link to instead?